package.json

```diff
@@ -19,7 +19,8 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.1",
-    "source-map-explorer": "^2.5.0"
+    "source-map-explorer": "^2.5.0",
+    "treetracker-web-map-core": "^1.1.0"
```

Comment:
@iib0011 i forgot to mark this one yesterday, you can delete this map-core package too